app/views/teaching_events/about_git_events.html.erb

```diff
@@ -0,0 +1,101 @@
+<%= render Campaign::HeaderComponent.new(
+  title: ["Get Into Teaching", "events"],
+  colour: "yellow",
+  image: "media/images/content/hero-images/0025.jpg"
+) %>
+
+<div class="subject-specific">
+  <article>
+    <section class="statement">
+      <span class="pink xlarge bold">What to expect</span>
+    </section>
+
+    <section>
+      <p>Whether you're ready to start your career in teaching or just curious, we can answer your questions at a Get Into Teaching event.</p>
+
+      <p>They're a series of relaxed and informative events giving you the opportunity to get expert advice and talk to teachers and local training providers.</p>
+
+      <h2 class="heading-m">At the event</h2>
+
+      <p>Drop in any time during an event for presentations, one-to-one chats or to meet others like you who are thinking of stepping into the classroom.</p>
+
+      <p>You can discover:</p>
+
+      <ul>
+        <li>if teaching is right for you</li>
+        <li>how to apply for teacher training</li>
+        <li>what funding is available</li>
+      </ul>
+
+      <h2 class="heading-m">Reserve your place</h2>
+
+      <p>You'll need to register to attend a Get Into Teaching event. Places are limited, so the sooner you sign up, the better.</p>
+    </section>
+
+    <section class="grey-bg fullwidth">
+      <%= render SubjectSpecific::TextBlockWithOneImageComponent.new(
+        heading: "Events in your region",
+        colour: "blue",
+        image_path: "media/images/content/homepage/here-to-help.jpg",
+        image_alt: "Child pointing something out to a teacher in the classroom.",
```

Comment:
Alt text needs updating @GDW7 

@gazcarr Are we changing this image?

Reply:
I think this image works fine here

Reply:
@ethax-ross Can we update the alt text as follows:


```suggestion
        image_alt: "A woman at a Get Into Teaching event wearing a blue T-shirt with Here to Help written on it.",
```